### DISCUSSION TITLE
Automatic deletion of cookies for one domain only. Everything else stays the same. How?

### DISCUSSION THREAD

**Discussion by vassilyzaitsev** [Cookie-AutoDelete/Cookie-AutoDelete#1321] [Q&A / Support]
I installed this plugin only because I would like to automatically delete cookies from one domain and its subdomains... But I wasn't able to figure out how... 

If I'm correct there's no option to do that right? Or there is? I can only delete automatically cookies from "all domains with exceptions"... 

I'd like to delete only for this particular annoying website.

Thanks.

**Comment by kennethtran93** [comment]
You are correct - we do not have a block list yet.  If cookie isn't needed for one particular site, you can block it from being created through the browser side.

**Reply by vassilyzaitsev** [reply]
> ou can block it from being created through the browser side

Hmmm I didn't know that. Thanks.

**Comment by conuaunoc** [comment]
Just use Private/Incognito mode for that domain
